develop-business/readme.md

```diff
@@ -0,0 +1,113 @@
+> Goal of this guide: Help you develop products and services to stimulate growth.
+
+# Business Development
+
+Every company needs to have a source of income that is equal to its expenses to avoid bankruptcy. The moment your income is higher, you can start growing.
+
+The job of a businesses developer is to develop and implement growth opportunities. This means that it is also your job to ensure there is ample income to create growth. This can be through selling products and services or finding funding and grants.
+
+There are two ways of generating income. Transactions based on current value and transactions based on future value.
+
+- Current value: Something that you offer that has a direct value to others. This usually is a product or service. These are transactions between supplier and the user.
+
+- Future value: Something that could be great someday. This can be anything, from a product to a movement or phenomenon. This is usually in the form of funding or grants.
+
+## Perspective is key when developing products and services
+
+You can not create growth without valuable products or services and you can't create those without talking to your target audience.
+
+The perceived value of your product or service directly determines your income. This can be the price a user  is willing to pay for your product, the future value of your company to an investor, the effect the government thinks your company might have on the environment, etc..
```

Comment:
"the effect the government thinks your company might have on the environment, etc.." 
This part seems to come out from a bit left field relative to the previous two points (which you had entered into already in the previous section). Is it important that its the government in any way in terms of grants for example? Or lobbying by activists could have a negative effect on the brand?